lib/appraisal/matrix/extensions/appraisal_file.rb

```diff
@@ -21,14 +38,18 @@ def appraisal_matrix(**kwargs, &block)
       #   [[a, x], [a, y], [a, z]]
       # ]
       names_and_versions_to_test =
-        kwargs.map do |gem_name, version_request|
-          if version_request.is_a?(Hash)
-            raise "TODO: Version request options not implemented yet"
-          else
-            minimum_version = Gem::Version.new(version_request)
-          end
+        kwargs.map do |gem_name, version_options|
+          version_array =
+            case version_options
+            when String
+              VersionArray.new(gem_name: gem_name, versions: [">= #{version_options}"])
```

Comment:
I might add a check here to ensure this is a compatible Gem version. I forsee someone doing:
`rails: ">= 5.0"` as the argument by accident.